object_manager_zone_auth.go

```diff
@@ -0,0 +1,50 @@
+package ibclient
+
+import "fmt"
+
+func (objMgr *ObjectManager) CreateZoneAuth(
+	fqdn string,
```

Comment:
The list of fields must be extended at least by 'view' (DNS view), and preferably by 'zone_format' (possible values in WAPI: FORWARD, IPV4, IPV6), with appropriate busyness-logic changes.

Reply:
I've added both DNS view and zone_format to the object and the applicable functions